src/optimize/bundles_route/bundles_route.js

```diff
@@ -0,0 +1,62 @@
+import { isAbsolute, extname } from 'path';
+
+import LruCache from 'lru-cache';
+
+import { createDynamicAssetResponse } from './dynamic_asset_response';
+
+/**
+ *  Creates a route that serves files from `bundlesPath`. If the
+ *  file is js or css then it is searched for instances of
+ *  PUBLIC_PATH_PLACEHOLDER and replaces them with `publicPath`.
+ *  @param {Object} options
+ *  @property {string} options.bundlesPath
+ *  @property {string} options.basePublicPath
+ *  @return {undefined}
+ */
+export function createBundlesRoute({ bundlesPath, basePublicPath }) {
+  const cache = new LruCache(100);
```

Comment:
Either too generically named or needs a comment, especially as it's not used in this file either. Maybe `fileHashCache` or something like that(?)